WebKitBrowser/CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ All notable changes to this RDK Service will be documented in this file.
 
 * For more details, refer to [versioning](https://github.com/rdkcentral/rdkservices#versioning) section under Main README.
 
+## [1.1.13] - 2023-03-22
```

Comment:
Please also update patch version in WebKitBrowser.cpp

Reply:
I updated the patch version. Thanks.